libs/types/src/feature.ts

```
@@ -3,11 +3,13 @@
  */
 
 import type { PlanningMode, ThinkingLevel, GitWorkflowSettings } from './settings.js';
+import type { SignalChannel } from './signal-channel.js';
 import type { ReasoningEffort } from './provider.js';
 import type { AgentRole } from './agent-roles.js';
 import type { WorkItemState } from './authority.js';
 import type { ReviewThreadFeedback, PendingFeedback } from './coderabbit.js';
 import type { PipelineState } from './pipeline-phase.js';
+import type { SignalChannel, SignalMetadata } from './signal-channel.js';
```

Biome (2.4.6)

[error] Shouldn't redeclare 'SignalChannel'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)